easybuild/easyconfigs/r/RFdiffusion/RFdiffusion-1.1.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -1,61 +1,95 @@
+    }),
+    ('executing', '1.2.0', {
+        'checksums': ['19da64c18d2d851112f09c287f8d3dbbdf725ab0e569077efb6cdcbd3497c107'],
+    }),
+    ('icecream', '2.1.3', {
+        'checksums': ['0aa4a7c3374ec36153a1d08f81e3080e83d8ac1eefd97d2f4fe9544e8f9b49de'],
     }),
     ('dllogger', '1.0.0', {
         'source_urls': ['https://github.com/NVIDIA/dllogger/archive/refs/tags'],
         'sources': ['v%(version)s.zip'],
         'checksums': ['07d0cd9b9b56f454f0c186a0889137e9f94e1979fca3d35911967c874c93c191'],
     }),
-    ('decorator', '5.1.1', {
-        'checksums': ['637996211036b6385ef91435e4fae22989472f9d571faba8927ba8253acbc330'],
+    ('pynvml', '11.5.0', {
+        'checksums': ['d027b21b95b1088b9fc278117f9f61b7c67f8e33a787e9f83f735f0f71ac32d0'],
     }),
-    ('se3-transformer', version, {
+    ('SE3Transformer', version, {
+        'modulename': 'se3_transformer',
         'source_tmpl': 'v%(version)s.tar.gz',
-        'source_urls': ['https://github.com/RosettaCommons/RFdiffusion/archive/refs/tags'],
+        'source_urls': ['https://github.com/RosettaCommons/RFdiffusion/archive/refs/tags/'],
         'start_dir': 'env/SE3Transformer',
         'checksums': ['57d82f0d43540c2912eda3f1d34ad90b13db14966ee069c427e217fe78f0297f'],
     }),
     (name, version, {
-        'patches': ['%(name)s-%(version)s_fix-find-packages.patch'],
-        'sources': {
-            'filename': '%(name)s-%(version)s.tar.gz',
-            'download_filename': 'v%(version)s.tar.gz',
-            'source_urls': ['https://github.com/RosettaCommons/RFdiffusion/archive/refs/tags'],
-        },
+        'patches': [
+            'RFdiffusion-1.1.0_setup.patch',
+            # change default paths of models and schedules to $RFDIFFUSION_MODELS and RFDIFFUSION_SCHEDULES
+            # to be set by dependencies RFDiffusion-models and RFDiffusion-schedules.
+            # Expect default config in  _pysp/rfdiffusion/config/inference:
+            'RFdiffusion-1.1.0_data_paths.patch',
+        ],
+        'source_urls': ['https://github.com/RosettaCommons/RFdiffusion/archive/refs/tags/'],
+        'sources': [{'filename': '%(name)s-%(version)s.tar.gz', 'download_filename': 'v%(version)s.tar.gz'}],
         'checksums': [
             {'RFdiffusion-1.1.0.tar.gz': '57d82f0d43540c2912eda3f1d34ad90b13db14966ee069c427e217fe78f0297f'},
-            {'RFdiffusion-1.1.0_fix-find-packages.patch':
-             'e25da7f476acacb6af58bd84f40aca712538900b0b48532b22f9a133d6886da2'},
+            {'RFdiffusion-1.1.0_setup.patch': '1d17fae9d4d3716814f0d9bb6587658590dc07772c87e92fb9ad81cda6a2539d'},
+            {'RFdiffusion-1.1.0_data_paths.patch': '3d1ddd1684004b4757958ef3d07b96a0f8976091fe670381fc723ad4eb6d65a5'},
         ],
     }),
 ]
 
-sanity_pip_check = True
+postinstallcmds = [
+    'cp -rpP config examples %s/%%(namelower)s' % _pysp,
+]
+
+sanity_check_commands = [
+    "run_inference.py --help",
+    "run_inference.py 'contigmap.contigs=[150-150]' " +
```

Comment:
This command will create two output directories in %(installdir)s, is there a way to avoid this?